docs/reference/manifest/desktopsettings.md

```diff
@@ -1,25 +1,40 @@
 ---
 title: DesktopSettings element in the manifest file
 description: ''
-ms.date: 10/09/2018
+ms.date: 01/13/2020
 localization_priority: Normal
 ---
 
 # DesktopSettings element
 
 Specifies source location and control settings that apply when your mail add-in is used on a desktop computer.
 
+> [!IMPORTANT]
```

Comment:
Is this note meant to imply that it is not available for other Outlook hosts? If so, then it should probably say that. If not, then I don't understand the point of the note. (In other words, should the word "only" be in front of "available"?

Reply:
Thanks for pointing that out. I included the word "only" in the others but forgot it for this one. Updated now.